### ISSUE TITLE
Non-standard field names for F3XN, F3N

### ISSUE THREAD

**Issue by sahilchinoy** [NYTimes/Fech#76]
Is there a reason for the difference in key names between responses for different F3 form types? For example, F3N filings get a `col_a_individual_contributions_itemized` field, while F3XN filings get a `col_a_individuals_itemized` field. I know the forms are different, but the fields seem similar enough to be standardized.

F3N | F3XN
--- | ---
`col_a_individual_contributions_unitemized` | `col_a_individuals_unitemized`
`col_a_total_individual_contributions` | `col_a_individual_contribution_total`
`col_a_political_party_contributions` | `col_a_political_party_committees`
`col_a_pac_contributions` | `col_a_other_political_committees_pacs`
`col_a_transfers_from_authorized` | `col_a_transfers_from_aff_other_party_cmttees`
`col_a_offset_to_expenditures` | `col_a_offsets_to_expenditures`

**Comment by dwillis** [comment]
It's complicated. Form 3 and Form 3X are for different kinds of committees, which is one reason for the variation in field names. Our policy is to adhere as closely to the field names on the form and not apply standardization beyond that, but I can see where cases of pluralization could be considered.

**Comment by sahilchinoy** [comment]
Makes sense. I'll try to maintain that table of similar field names, let me know if you might find it helpful. Thanks for all your work on this.
